adcm_aio_client/core/utils.py

```diff
@@ -1,4 +1,4 @@
-from typing import Awaitable, Iterable
+from collections.abc import Awaitable, Iterable
 import asyncio
 
 from adcm_aio_client.core.types import RequesterResponse
```

Comment:
can `# noqa: F821` and `TODO` below  in this file be removed now?

Reply:
yes, removed